config/webpack.config.js

```diff
@@ -71,6 +71,12 @@ module.exports = env => {
   if (env && env.pluginName !== undefined) {
     var pluginEntries = {};
     pluginEntries[env.pluginName] = plugins['entries'][env.pluginName];
+    for (var entry in plugins['entries']) {
+      if (entry.startsWith(env.pluginName + ":")) {
```

Comment:
Could you do `entry.startsWith(env.pluginName)` and then remove line 73?

Reply:
Thinking out loud here. Wouldn't that potentially also add something like `plugin_something` for just `plugin`? More concrete, let's say we have `remote_execution` plugin which also has `remote_execution:base`. If we're building assets for a separate plugin `remote_execution_ansible` which has a build dependency on `remote_execution` then I wonder if this could be an issue. It's unlikely because you'd probably build `remote_execution` first without `remote_execution_ansible` installed and while building `_ansible` the if condition would be false. Still, the current version feels more correct.

Reply:
Yep, @sharvit the ':' separator was meant to prevent other plugin's assets from being compiled when something more generic is passed in.  Imagine trying to compile 'foreman' assets and having any other plugin that happens to start with 'foreman' compiled.

Reply:
Good point @ekohl